elixir/README.md

```diff
@@ -7,10 +7,11 @@ The applications in this directory make use of the [AtomVM Mix Plugin](https://g
 
 Some example programs can only run on specific platform.  The following table summarizes the platforms on which the Elixir example programs can be run.
 
-| Example Program | esp32 | stm32 | generic_unix |
-|-----------------|-------|-------|--------------|
-| Blinky          | ✅ | ✅ | ❌ |
-| HelloWorld      | ✅ | ✅ | ✅ |
+| Example Program | esp32 | stm32 | Pico | Pico W | generic_unix |
+|-----------------|-------|-------|------|--------|--------------|
+| Blinky          | ✅ | ✅ | ❌ | ❌ | ❌ |
```

Comment:
Needs update for STM32 Elixir Blinky to work OOTB.

Reply:
Fixed, but not showing as outdated for some reason.

Reply:
Funny, still show up that way, but no big deal. We should have the Fixes in by tomorrow, no point in you staying up late to review them tonight ;)